apps/explorer/src/components/ownedobjects/views/UnderlyingObjectCard.tsx

```diff
@@ -42,16 +54,20 @@ export function UnderlyingObjectCard({
     }
 
     const fieldsData = getObjectFields(data);
-    return fieldsData ? (
-        <SyntaxHighlighter
-            code={JSON.stringify(
+    // Return null if there are no fields
+    if (!fieldsData || !normalizedStruct?.fields) {
+        return null;
+    }
+    return (
+        <FieldItem
```

Comment:
Probably we need to <FieldItem /> here for the case we want to display the name and value